test/test-utils/test-utils.ts

```diff
@@ -126,6 +126,7 @@ export function createTestClient(): MatrixClient {
         mxcUrlToHttp: (mxc) => `http://this.is.a.url/${mxc.substring(6)}`,
         setAccountData: jest.fn(),
         setRoomAccountData: jest.fn(),
+        setRoomReadMarkers: jest.fn().mockResolvedValue({}),
```

Comment:
linter seems to be complaining about this because it's defined 2 lines lower

Reply:
```suggestion
```